app/src/main/AndroidManifest.xml

```diff
@@ -122,6 +122,9 @@
     <activity android:name=".testing.HtmlParserTestActivity" />
     <activity android:name=".testing.HomeTestActivity" />
     <activity android:name=".testing.InputInteractionViewTestActivity" />
+    <activity
+      android:name=".testing.FontScaleTestActivity"
```

Comment:
This activity is not in alphabetical order.

Reply:
Updated.